.vscode/launch.json

```diff
@@ -236,14 +236,19 @@
             "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
             "stopOnEntry": false,
             "env": {
+                // "FLUID_TEST_DRIVER": "tinylicious",
                 "TS_NODE_PROJECT": "${fileDirname}/tsconfig.json",
             },
-            "args": ["--no-timeouts", "--exit", "--require", "ts-node/register","${file}", ],
-            "cwd": "${workspaceFolder}",
+            "args": [
+                "--no-timeouts",
+                "--exit",
+                "--require", "ts-node/register",
+                "${file}", ],
+            "cwd": "${fileDirname}",
             "skipFiles": [
                 "<node_internals>/**/*.js"
             ],
-            "preLaunchTask": "fluid-build $cwd --nolint"
+            "preLaunchTask": "fluid-build $cwd -s tsc"
```

Comment:
changed this to make it a bit faster, we don't need to build esnext, or the tests for dependencies